recipes/irfinder/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: irfinder
+  version: '1.2.2'
+  
+build:
+  number: 0
+  
+source:
+  git_rev: v1.2.2
+  url: https://github.com/williamritchie/IRFinder/archive/v1.2.2.tar.gz
+  md5: bf5d672b7ff61feaf24d1299b87fa2f1
+  
+requirements:
+  build:
+    - gcc
```

Comment:
Indeed I missed this one. I'm relatively new to colloboration with github. What would be the best/cleanest way to make the changes (ie change on my local branch, push to my account, do a new pull request, or make the changes directly on github) ?